gcloud/bigtable/row_filters.py

```diff
@@ -119,7 +119,7 @@ class _RegexFilter(RowFilter):
     """
 
     def __init__(self, regex):
-        self.regex = regex
+        self.regex = _to_bytes(regex)
```

Comment:
You made me think about this a bit more. In reality, the only valid input is `bytes`, so the conversion should happen in the constructor to normalize to something valid.